BREAKING.md

```diff
@@ -24,6 +25,9 @@ Move the type from the merge-tree package where it isn't used to the sequence pa
 ## Remove logger property from IContainerContext
 The logger property in IContainerContext became an optional parameter in [release 0.56](#Set-logger-property-as-optional-parameter-in-IContainerContext). This property has now been removed. The `taggedLogger` property is now set as a required parameter in `IContainerContext` interface.
 
+## Set raiseContainerWarning property as optional parameter on IContainerContext
+`raiseContainerWarning` is set as an optional parameter on `IContainerContext` interface and would be removed from `IContainerContext` interface and `ContainerContext` class in the next release. Please see [#Remove-raiseContainerWarning-property] for more details.
```

Comment:
Is setting this as an optional parameter breaking change?

Reply:
Since the `common/lib/container-definitions/api-report/container-definitions.api.md` is modified, it seems like a change that should be added to breaking.md

Reply:
Breaking changes imply that consumer of this API needs to make an explicit action/change around API to avoid build failures and/or behavior regressions. It does not seem that is the case here.